src/resource_server/views/api_v1/api.py

```diff
@@ -15,14 +15,36 @@ def cmd(endpoint):
     path_file = current_app.config["CMD_PATH_FILE"]
     base_url = current_app.config["SSH_KEYSIGN_BASE_URL"]
     user = current_user.claims[current_app.config["SSH_PRINCIPAL_CLAIM"]]
+    allowed_params = current_app.config["API_ALLOWED_QUERY_PARAMS"]
```

Comment:
so there is one set of allowed query params for all endpoints ?
and it has to be configured via settings.py ?

Reply:
yes pretty much, unless we use the config file to get the allowed_query params per endpoint.

Reply:
ok ... then why do we have the parameters section in the config file if we are not using it?
https://github.com/ternaustralia/rs-remote-job/blob/develop/config/config.json#L15

Reply:
As I understand those parameters are different and those are part of the backend but not parameters that come from the api.

Reply:
ok ... then why do these parameters have a default value but no value ?

Reply:
as I said I thought it was part just the backend and I never validated that default because I haven't see that to validate but now is different I'll change no problem.

Reply:
a default is not really there to validate, but rather to replace a missing value.
so for the global section the question is rather ...
- Where would values come from to replace defaults ?
similar for exec parameters 
- Where would values come from to replace defaults ?

so far the examples I have seen for the global parameters section, is that these values are much more like deployment configuration and better not supplied as request parameter... so where would values then come from ?

the exec parameters however look very much like parameters to commands to execute ... they would fit the idea of a request parameter very well.